package.json

```diff
@@ -53,6 +53,7 @@
     "@react-native-community/eslint-config": "^2.0.0",
     "@release-it/conventional-changelog": "^2.0.0",
     "@types/jest": "^26.0.0",
+    "@types/node": "^14.14.37",
```

Comment:
Sorry if I'm missing something obvious, but I cannot find where these types are used?

Types in the `@types/node` package are only available when running inside Node.js, not in the browser which this new code is targeting, right?